tests/test_api.py

```diff
@@ -6,7 +6,7 @@
 from nose.tools import assert_equal
 
```

Comment:
Got `ImportError: Entry point ('console_scripts', 'openfisca serve') not found` 
at `api576[openfisca-france]$ nosetests tests/test_api.py`